azure-iot-device/samples/pnp/pnp_helper.py

```diff
@@ -0,0 +1,40 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
Can pnp_helper and pnp_methods be combined?

Reply:
after trying to put everything in 1 file.....i decided to keep a distinction between 2 files...one is the thin layer over device SDK functionality....the other is pure helper methods...

---
In reply to: [408397435](https://github.com/Azure/azure-iot-sdk-python/pull/535#discussion_r408397435) [](ancestors = 408397435)